aal_purchase_form/README.rst

```diff
@@ -0,0 +1,79 @@
+=================
+aal_Purchase_Form
+=================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faal-lightgray.png?logo=github
+    :target: https://github.com/OCA/aal/tree/12.0-fix-aal_purchase_form/aal_purchase_form
+    :alt: OCA/aal
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/aal-12-0-fix-aal_purchase_form/aal-12-0-fix-aal_purchase_form-aal_purchase_form
+    :alt: Translate me on Weblate
+
+|badge1| |badge2| |badge3| |badge4| 
+
+This module create new form in purchase_order.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module you need to:
+
+#. Got to *Purchase> select purchase order > Print*
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/aal/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/aal/issues/new?body=module:%20aal_purchase_form%0Aversion:%2012.0-fix-aal_purchase_form%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Ecosoft
+
+Contributors
+~~~~~~~~~~~~
+
+* Jirawat <jirawata@ecosoft.co.th>
```

Comment:
Jirawat <jirawata@ecosoft.co.th> and Saran Lim <saranl@ecosoft.co.th>
Not Contributors this module.